performance-tools/docs/performance-test-cluster-setup.md

```diff
@@ -0,0 +1,108 @@
+# Performance test setup
+
+## Installation of Load Generating Cluster.
+
+### Pre-requisites
+
+We have used GKE for installing the load generator cluster. Following are the prerequisites for the same.
+
+1. Create GKE account with `owner` rights
+2. Have domain name to be used.
+
+### Installation
+
+#### 1. Installation of the cluster
+We are using kyma for the installation as it provides authentication and authorization. Additionally we can leverage grafana which comes with Kyma. To install the cluster follow the steps documented [here](https://kyma-project.io/docs/root/kyma/#installation-install-kyma-on-a-cluster).
+
+
+#### 2. Installation of InfluxDB
+k6 uses InfluxDB to store results and Grafana could be used to visualize the test results. To install InfluxDB we used the application available on [GCP market place](https://console.cloud.google.com/marketplace/details/google/influxdb?q=influxdb). The version of influxDB has been backported to [1.6](gcr.io/cloud-marketplace/google/influxdb@sha256:23d3f92f3f375a7e37ee4e54e739a068e9cf80a570ffecce60b97076c15855b6`), since 1.7 has issues with random timeouts/freeze as mentioned [here](https://github.com/influxdata/influxdb/issues/12731) 
+
+
+
+#### 3. Configuration
+
+Following are the configurations required:
+1. InfluxDB
+   * Login into influx:
+     ```bash
+     influx -username <>  -password <>
+     ```
+   * Create USER to readUser and writeUser
+     ```bash
+      CREATE USER readUser with PASSWORD <>
+      CREATE USER writeUser with PASSWORD <>
+     ```
+   * Create database
+     ```bash
+     CREATE DATABASE "database"
+     ```
+   * GRANT appropriate rights to users
+      ```bash
+      GRANT ALL ON "database" TO "writeUser"
+      GRANT READ ON "database" TO "readUser"
+      ```
+2. Grafana
```

Comment:
Remove

Reply:
What should I remove, 2.Grafana ?

Reply:
same reason as above. We want to have a documentation on how to install the load-generator-cluster